net-irc/inspircd/inspircd-2.0.25.ebuild

```diff
@@ -20,7 +20,7 @@ RDEPEND="
 	geoip? ( dev-libs/geoip )
```

Comment:
Any reason not to `Closes:` that bug instead?

Reply:
Nope, changed.